draft-ietf-ppm-dap.md

```diff
@@ -358,10 +358,27 @@ client could not report 10^6s or -20s.
 
 # Message Transport
 
-Communications between DAP entities are carried over HTTPS {{!RFC2818}}. HTTPS
-provides server authentication and confidentiality. When client authenticaiton
-is also required, the client uses the mechanism described in
-{{https-sender-auth}}.
+Communications between DAP participants are carried over HTTPS {{!RFC9110}}.
+HTTPS provides server authentication and confidentiality. Use of HTTPS is
+REQUIRED.
+
+## HTTPS Request Authentication {#request-authentication}
+
+DAP is made up of several sub-protocols in which different subsets of the
+protocol's participants interact with each other.
+
+In those cases where a channel between two participants is tunneled through
+another protocol participant, DAP mandates the use of authenticated encryption
```

Comment:
Bump: We're not using HPKE for authenticated encryption here. The properties we need are confidentiality and non-malleability.
```suggestion
another protocol participant, DAP mandates the use of public-key encryption
```